webapp/channels/src/components/emoji_picker/components/emoji_picker_category.tsx

```diff
@@ -38,20 +35,15 @@ function EmojiPickerCategory({category, categoryRowIndex, selected, enable, onCl
     });
 
     return (
-        <OverlayTrigger
-            trigger={['hover', 'focus']}
-            delayShow={Constants.OVERLAY_TIME_DELAY}
+        <WithTooltip
+            id={`emojiPickerCategoryTooltip-${category.name}`}
             placement='bottom'
-            overlay={
-                <Tooltip
-                    id='skinTooltip'
-                    className='emoji-tooltip'
-                >
-                    <FormattedMessage
-                        id={`emoji_picker.${category.name}`}
-                        defaultMessage={category.message}
-                    />
-                </Tooltip>
+            className='emoji-tooltip'
```

Comment:
Why do we need a new class name? `WithTooltip` should already handle all versions of the tooltip approved by UX, and adding a `classname` opens the path to inconsistencies in the future.

Reply:
Yeah i didnt wanted use case here that tooltip needed a lot more of z-index because we are showing those in the popover. Do you think we should have that 9999 z-index by default in the tooltip styling by default?

Reply:
I see no reason why not. Do we ever want the tooltip to be behind something?

Reply:
Nothing i can think of right now

Reply:
Done